lib/rubocop/cop/lint/void.rb

```diff
@@ -6,6 +6,15 @@ module Lint
       # Checks for operators, variables, literals, lambda, proc and nonmutating
       # methods used in void context.
       #
+      # `each` blocks are allowed to prevent false positives.
```

Comment:
You might expand on this a bit here - e.g. saying that the receiver might be an enumerator.